Makefile

```diff
@@ -22,4 +22,4 @@ vendor: Puppetfile
 
 .PHONY: all_diffs
 all_diffs:
-	./bin/octocatalog-diff
+	@./bin/octocatalog-diff
```

Comment:
Now that we're not using `make` for the output should we still add the `@` directive?

Reply:
Yeah, good point, I'll change this back since this is really only here for people to run manually if they would like to.